common/src/main/java/bio/terra/catalog/datarepo/DatarepoService.java

```diff
@@ -148,7 +148,7 @@ private SnapshotPreviewModel getPreviewTable(String snapshotId, String tableName
       UUID id = UUID.fromString(snapshotId);
       return datarepoClient
           .snapshotsApi()
-          .lookupSnapshotPreviewById(id, tableName, null, maxRows, null, null);
+          .lookupSnapshotPreviewById(id, tableName, null, maxRows, null, null, null);
```

Comment:
This API is now deprecated, I think this replacement will work:
```suggestion
          .querySnapshotDataById(id, tableName, new QueryDataRequestModel().limit(maxRows));
```